File: projects/library/php/composer/{{cookiecutter.project_slug}}/tests/HelloTest.php
<?php

declare(strict_types=1);

namespace {{ cookiecutter.php_namespace }}\Tests;

use PHPUnit\Framework\TestCase;
use {{ cookiecutter.php_namespace }}\Hello;

final class HelloTest extends TestCase
{
    public function testDefault(): void
    {
        $this->assertSame('hello, world', Hello::greet());
    }

    public function testName(): void
    {
        $this->assertSame('hello, ana', Hello::greet('ana'));
    }
}

File: projects/library/go/module/{{cookiecutter.project_slug}}/hello_test.go
package {{ cookiecutter.package_name }}

import "testing"

func TestHello(t *testing.T) {
	if got := Hello(""); got != "hello, world" {
		t.Fatalf("Hello(\"\") = %q", got)
	}
	if got := Hello("ana"); got != "hello, ana" {
		t.Fatalf("Hello(ana) = %q", got)
	}
}
